examples/example_util.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import numpy
+
+import plotting
+from paysage import batch
+from paysage import backends as be
+
+# ----- DEFALT PATHS ----- #
+
+def default_paysage_path():
+    try:
+        # base on script location (eg from import or command line script)
+        paysage_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    except NameError:
+        # Called from outside of script or import (eg within ipython, idle, or py2exe)
+        paysage_path = os.path.dirname(os.getcwd())
+    return(paysage_path)
+
+def default_mnist_path(paysage_path = None):
+    if not paysage_path:
+        paysage_path = default_paysage_path()
+    return os.path.join(paysage_path, 'mnist', 'mnist.h5')
+
+def default_shuffled_filepath(paysage_path):
+    shuffled_filepath = os.path.join(paysage_path, 'mnist', 'shuffled_mnist.h5')
+    if not os.path.exists(shuffled_filepath):
+        print("Shuffled file does not exist, creating a shuffled dataset.")
+        shuffler = batch.DataShuffler(filepath, shuffled_filepath, complevel=0)
```

Comment:
It looks like `filepath` is not yet defined here. The path to the mnist file should probably be passed in as an argument.